packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.base.tsx

```diff
@@ -191,7 +194,7 @@ export class ColorPickerBase extends React.Component<IColorPickerProps, IColorPi
                 <td>{textLabels.r}</td>
                 <td>{textLabels.g}</td>
                 <td>{textLabels.b}</td>
-                {!alphaSliderHidden && <td>{textLabels.a}</td>}
+                {!alphaSliderHidden && <td>{useTransparencySlider ? props.alphaLabel : textLabels.a}</td>}
```

Comment:
Lets not change here. Lets update textLabel.a in the constructor as required

Reply:
can't change textLabel.a in the constructor based on props value since its initialized only once.